File: Harvard_CS50_Introduction_to_CS/CS50-Pset6-master/cash.py
# import get_float from cs50
from cs50 import get_float
user_money = 0
# get positive amount of money from user
while (user_money <= 0):
    user_money = get_float("Change owned: ")
# convert money into coin unit
u = user_money * 100
# converting to number of coins
num_coins = u//25 + (u % 25)//10 + ((u % 25) % 10)//5 + ((u % 25) % 10) % 5
print(int(num_coins))

File: Harvard_CS50_Introduction_to_CS/CS50-Pset1-master/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    //int quarters = 25, dimes = 10, nikels = 5, pennies = 1;
    float dollars = 0;
    while (!(dollars > 0))
    {
        dollars = get_float("change owned: ");
    }
    //convert dollars to cents
    int cents = round(dollars * 100);
    int coins = cents/25 + (cents%25)/10 + ((cents%25)%10)/5 + (((cents%25)%10)%5)/1;
    printf("%i\n",coins);
}
